api/camunda/camunda-openapi.yaml

```diff
@@ -495,8 +495,8 @@ paths:
       description: |
         Get the user task by the user task key.
 
-        Note that this endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
-        The Camunda 8 API (REST) Overview page provides further details.
+        Note that this endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and not enabled on Camunda clusters out of the box.
+        The [Camunda 8 API (REST) Overview page](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api) provides further details.
 
         :::note
         This endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and may be subject to change
```

Comment:
After this merges, I'll go into the upstream spec and remove these remaining links. They will be injected during the generation, just like the two links in the diff above this comment.